Add GitHub Actions workflow to run runbook

lourens-octopus opened the pull request.

lourens-octopus committed:
Add GitHub Actions workflow to run runbook

octopus-bot-account commented:
Your PR has been deployed to test system: [http://dad-joker-2000-pr-111.australiaeast.cloudapp.azure.com](http://dad-joker-2000-pr-111.australiaeast.cloudapp.azure.com)

lourens-octopus committed:
Add runbook execution step in workflow

lourens-octopus committed:
Remove test and production environment steps

Removed steps for creating and deploying test and production environments in the workflow.

lourens-octopus committed:
Fix environments configuration in workflow file

lourens-octopus committed:
Update create-ephemeral-environment.yml
